prow/plugins/trigger/trigger.go

```diff
@@ -119,42 +123,53 @@ func getClient(pc plugins.PluginClient) client {
 }
 
 func handlePullRequest(pc plugins.PluginClient, pr github.PullRequestEvent) error {
-	trustedOrg, joinOrgURL := trustedOrgForRepo(pc.PluginConfig, pr.Repo.Owner.Login, pr.Repo.Name)
-	return handlePR(getClient(pc), trustedOrg, joinOrgURL, pr)
+	org, repo, _ := orgRepoAuthor(pr.PullRequest)
+	return handlePR(getClient(pc), pc.PluginConfig.TriggerFor(org, repo), pr)
 }
 
 func handleIssueComment(pc plugins.PluginClient, ic github.IssueCommentEvent) error {
-	trustedOrg, _ := trustedOrgForRepo(pc.PluginConfig, ic.Repo.Owner.Login, ic.Repo.Name)
-	return handleIC(getClient(pc), trustedOrg, ic)
+	return handleIC(getClient(pc), pc.PluginConfig.TriggerFor(ic.Repo.Owner.Login, ic.Repo.Name), ic)
 }
 
 func handlePush(pc plugins.PluginClient, pe github.PushEvent) error {
 	return handlePE(getClient(pc), pe)
 }
 
-// trustedOrgForRepo returns the configured trusted organization and a URL for it
-// for the provided org and repo combination.
-func trustedOrgForRepo(config *plugins.Configuration, org, repo string) (string, string) {
-	if trigger := config.TriggerFor(org, repo); trigger != nil && trigger.TrustedOrg != "" {
-		return trigger.TrustedOrg, trigger.JoinOrgURL
+// trustedUser returns true if user is trusted in repo.
+//
+// Trusted users are either repo collaborators, org members or trusted org members.
+// Whether repo collaborators and/or a second org is trusted is configured by trigger.
+func trustedUser(ghc githubClient, trigger *plugins.Trigger, user, org, repo string) (bool, error) {
```

Comment:
Why pass around the entire trigger config when we just need the org?

Reply:
We're using `OnlyOrgMembers` `TrustedOrg` and `JoinOrgURL` in various places (this method needs the first two).  The `(string, string)` return value was a pain to work with.